src/Cake.Common/Tools/OctopusDeploy/OctopusDeployRunner.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+using Cake.Core.Utilities;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    /// <summary>
+    /// The OctopusDeploy runner
+    /// </summary>
+    public sealed class OctopusDeployRunner : Tool<CreateReleaseSettings>
```

Comment:
I thought it would be logicial to group both CreateRelease and Deploy in the same runner since they invoke the same exe.  Deploy is another issue, which I plan on doing next.  Would it make more sense if I included both in the same pull request ?